cruise-control-metrics-reporter/src/test/java/com/linkedin/kafka/cruisecontrol/metricsreporter/CruiseControlMetricsReporterTest.java

```diff
@@ -202,4 +205,33 @@ public void testUpdatingMetricsTopicConfig() throws ExecutionException, Interrup
     topicDescription = adminClient.describeTopics(Collections.singleton(TOPIC)).values().get(TOPIC).get();
     assertEquals(2, topicDescription.partitions().size());
   }
+
+  @Test
+  public void testGetKafkaBootstrapServersConfigure() {
+    // Test with a "listeners" config with a host
+    Map<Object, Object> brokerConfig = buildBrokerConfigs().get(0);
+    Map<String, Object> listenersMap = Collections.singletonMap(KafkaConfig.ListenersProp(), brokerConfig.get(KafkaConfig.ListenersProp()));
+    String bootstrapServers = CruiseControlMetricsReporter.getBootstrapServers(listenersMap);
+    String urlParse = "\\[?([0-9a-zA-Z\\-%._:]*)]?:(-?[0-9]+)";
+    Pattern compile = Pattern.compile(urlParse);
```

Comment:
Nit: how about renaming this variable as `urlParsePattern`. `compile` seems to be too vague to me